src/ign-frontend/src/components/Demo.vue

```diff
@@ -117,7 +117,7 @@ export default Vue.component('Demo', {
   },
   data() {
     return {
-      apiUrl: 'https://ign-api.schrodinger-hat.it/v1',
+      apiUrl: 'https://ign-api.schroedinger-hat.org/v1',
```

Comment:
This and other occurrences are also being handle by #177

Reply:
I'll wait the merge and than I'll do a rebase

Reply:
I think there won't be any needs for a rebase, but thanks! Thé PR is merged now

Reply:
Rebase is always need :pray: ALWAYS